### ISSUE TITLE
Read dimension of the file on ftp server

### ISSUE THREAD

**Issue by umbertogenova** [blackcodetavern/ESP32_FTPClient#5]
Hi, there is an command for read dimension of the file.txt on the server ftp?
Thanks
Umberto

**Comment by ldab** [comment]
Yes, https://github.com/blackcodetavern/ESP32_FTPClient/blob/be5acd64822282e74b14ce4faf7b36b9c57b6a16/ESP32_FTPClient.cpp#L248
